contracts/GuardiansRegistration.sol

```diff
@@ -70,11 +68,12 @@ contract GuardiansRegistration is IGuardiansRegistration, ContractRegistryAccess
 	function unregisterGuardian() external onlyRegisteredGuardian onlyWhenActive {
 		delete orbsAddressToEthereumAddress[guardians[msg.sender].orbsAddr];
 		delete ipToGuardian[guardians[msg.sender].ip];
+		Guardian memory guardian = guardians[msg.sender];
```

Comment:
why memory and not storage? you may emit and then delete.

Reply:
1. I prefer to emit only after the work is done.
2. It may be cheaper to read in bulk than to ready the fields one by one using a storage pointer, I think. Not that it really matters in this scenario..